### DISCUSSION TITLE
群聊消息中，如何设置仅处理@机器人的消息？

### DISCUSSION THREAD

**Discussion by LingyiChen-AI** [simple-robot/Discussions#89] [Q&A] [answered]
### 涉及的编程语言

Java

### 涉及的组件库

mirai (QQ)

### 疑问描述

群聊消息中，如何设置仅处理@机器人的消息？

**Comment by ForteScarlet** [comment, marked as answer]
https://github.com/orgs/simple-robot/discussions/76
